README.adoc

```diff
@@ -1736,6 +1736,23 @@ Bundler.require(platform)
 Do not remove the `Gemfile.lock` from version control.
 This is not some randomly generated file - it makes sure that all of your team members get the same gem versions when they do a `bundle install`.
 
+== Testing
+
+=== Integration Testing
+
+Prefer integration style controller tests over functional style controller tests, https://api.rubyonrails.org/classes/ActionController/TestCase.html[as recommended in the Rails documentation].
```

Comment:
Maybe pin to a version to avoid broken links when Rails decides to phase the disgraced one out? https://api.rubyonrails.org/v7.0.0/classes/ActionController/TestCase.html

Reply:
Good idea, updated.